examples/nextjs/src/graphql/codegen.example.yml

```diff
@@ -0,0 +1,11 @@
+schema:
+  - YOUR-GRAFBASE-GRAPHQL-URL-HERE/graphql:
```

Comment:
it's not possible to use an env var here I assume?

Reply:
https://www.graphql-code-generator.com/docs/config-reference/codegen-config#environment-variables

Reply:
Nice, done